src/setStateIndicator/setIsSick.test.js

```diff
@@ -0,0 +1,37 @@
+import { setIsSick } from "./setIsSick";
+
+jest.useFakeTimers();
+
+describe("setIsSick", () => {
+  it("calls callback", () => {
+    const callback = jest.fn();
+    jest.spyOn(global.Math, "random").mockReturnValue(0.1);
+    setIsSick(callback);
+
+    jest.advanceTimersByTime(5000);
```

Comment:
instead of 5000 we could use the same constant that's used in the app. This way if you ever change the constant's value to, say, `7000`, your tests will get this change and should still pass, because nothing else but the timeout changed

Reply:
ok